src/components/Section/index.js

```diff
@@ -0,0 +1,37 @@
+import classnames from "classnames";
+import PropTypes from "prop-types";
+import { useFormContext } from "react-hook-form";
+import InputWrapper from "../InputWrapper";
+
+const Section = ({ fieldData, name, wrapClassName, wrapId }) => {
+  const { cssClass } = fieldData;
+
+  const {
+    formState: { errors },
+  } = useFormContext();
+
+  return (
+    <InputWrapper
+      errors={errors?.[name] || {}}
+      inputData={fieldData}
+      wrapClassName={wrapClassName}
+      wrapId={wrapId}
+    >
+      <div
+        className={classnames("gravityform__field__section", cssClass)}
+        style={{ borderBottom: "1px solid #ccc", marginBlock: "10px" }}
```

Comment:
@Robin-Riggs the markup of GF form doesn't match with yours. Please don't render any css styles there, also check for right markup, for example there should be <h3> tag rendered with `gsection_title` class and field label